zerver/lib/events.py

```diff
@@ -575,6 +579,7 @@ def apply_event(state: Dict[str, Any],
 
             if event['property'] == 'plan_type':
                 # Then there are some extra fields that also need to be set.
+                state['plan_includes_wide_organization_logo'] = event['value'] != Realm.LIMITED
```

Comment:
hm, it's not great that this check is replicated. Not familiar with the code enough to know the right way to handle it though.

Reply:
> ideally this check would not be replicated.

I didn't get what you're saying, could you please elaborate a bit about what check is being replicated here?

Reply:
oh, it just seems like there are two places in this file where we do the logic of 
`plan_includes_wide_organization_logo = plan_type != Realm.LIMITED`
and I would have expected that logic to only appear once.

I'm not familiar with this code though, so maybe Tim will have a different opinion.

Reply:
Oh, that's how it's supposed to be. I mean we could make a function and try extracting the logic, but IMHO, that's just really going out of our way in this case.